lib/agent/providers/hardware/windows.js

```diff
@@ -74,6 +75,8 @@ exports.get_firmware_info = function(callback) {
 
   var fetch = function(key, section, value){
     wmic.get_value(section, value, null, function(err, res){
+      if (!err && key == "device_type")
+        res ? res = 'Laptop' : res = 'Desktop'
```

Comment:
You could simplify:

`res = res ? 'Laptop' : 'Desktop'`

Also:

`key == 'device_type'` (single quotes)